runtime/Go/antlr/common_token_stream.go

```diff
@@ -154,7 +154,7 @@ func (c *CommonTokenStream) GetTokens(start int, stop int, types *IntervalSet) [
 		stop = len(c.tokens) - 1
 	}
 
-	for i := start; i < stop; i++ {
+	for i := start; i <= stop; i++ {
```

Comment:
are you sure? is it not related to discussion of open vs. closed interval? #1978

Reply:
I think you're right.  In general it's better for the start/stop to be inclusive.  All of the code I've seen using this is always doing a +1 here to make the stop work correctly.

Reply:
My API functions all assume inclusive intervals so `<=` seems right, doesn't it?

Reply:
This line is wrong if you're looking at the rest of the Go Interval implementation.  This change would be right in context of the Java API implementation (and the docs for this function above).